src/app/search/basic-search/basic-search.component.scss

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2022 OICR and UCSC
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+@use '@angular/material' as mat;
+@import '/src/materialColorScheme.scss';
+
+// Panel spacing
+.mat-expansion-panel-spacing {
+  margin: 0px;
+}
+
+// Panel header
+mat-expansion-panel mat-expansion-panel-header {
+  padding: 10px;
```

Comment:
This may have since changed, but the style guide recommends against pixels and to use rems. Are you using pixelst because you're overriding default Angular classes?

Reply:
I am overriding default Angular styles, but I used pixels mainly to follow Zeplin's specs. I can convert them to rems but just a bit on the fence about it since on my computer/browser 1rem = 10px for some reason... which is not the standard 16px. I've wondered if its because I'm on linux since my windows laptop follows the standard. I'll update this PR, can you confirm it displays nicely on your browser once its up?